Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
I will debug and check datagrid-bundle code and say more specifically, what causes this problem.

If we pass `ChoiceType::TYPE_EQUAL` we will have the folowing situation:
<img width="1250" alt="2017-03-12 19 45 03" src="https://cloud.githubusercontent.com/assets/1560126/23830702/a4e9dbb2-075c-11e7-976e-0a175f64f598.png">

When we pass `null` there we will have
<img width="1235" alt="2017-03-12 19 47 31" src="https://cloud.githubusercontent.com/assets/1560126/23830707/c239c768-075c-11e7-99b2-1af53d243ed9.png">

And a few lines above - in list mode we already have such line:
https://github.com/sonata-project/SonataClassificationBundle/blob/3.x/Controller/CategoryAdminController.php#L45